.github/ct.yaml

```diff
@@ -9,7 +9,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 
-helm-extra-args: --timeout 7200
+# consider helm install to be failed after 6 minutes
+helm-extra-args: --timeout 360
```

Comment:
I think giving this a few minutes more is ok … something around 15 minutes sounds reasonable to me.

Reply:
We are talking about installing a single chart here, aren't we? I wonder what chart would need that long to install. Can we meet in the middle, e.g. 10 minutes? We can increase once we have our first chart that doesn't complete installation in time ...